.github/workflows/release.yaml

```diff
@@ -0,0 +1,32 @@
+# .github/workflows/release.yml
+name: release
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
I think we should only allow releases on the `main` branch. Could this be restricted here?

Reply:
I added it in line 15, if I would add branch name here this action will be triggered also by pushing changes into main branch. This options work separately